html/saveForLater.html

```diff
@@ -1,75 +1,78 @@
 <!DOCTYPE html>
 <html>
-    <head>

```

Comment:
This is a big red just because I redid the formatting of your code, nothing was deleted.